todo/src/reducers/index.js

```diff
@@ -0,0 +1,60 @@
+import {ADD_TODO, TOGGLE_COMPLETED, DELETE_ITEM} from '../actions';
+
+let initialState;
+if (localStorage.getItem('state')) {
+  initialState = JSON.parse(localStorage.getItem('state'));
+} else {
+  initialState = {
+    todos: [
+      {
+        task: 'walk the dog',
+        complete: false,
+        id: 0,
+      },
+      {
+        task: 'do something else',
+        complete: false,
+        id: 1,
+      },
+    ],
+  };
+}
+
+export default (state = initialState, action) => {
```

Comment:
Good job on getting all of your cases working. The toggle complete is really clean.